src/client/cmd/mount.cpp

```diff
@@ -59,29 +59,24 @@ mp::ReturnCode cmd::Mount::run(mp::ArgParser* parser)
         return parser->returnCodeFrom(ret);
     }
 
-    auto on_success = [](mp::MountReply& reply) {
+    mp::AnimatedSpinner spinner{cout};
+
+    auto on_success = [&spinner](mp::MountReply& reply) {
+        spinner.stop();
         return ReturnCode::Ok;
     };
 
-    auto on_failure = [this, &parser](grpc::Status& status) {
-        auto ret = standard_failure_handler_for(name(), cerr, status);
-        if (!status.error_details().empty())
-        {
-            mp::MountError mount_error;
-            mount_error.ParseFromString(status.error_details());
-
-            if (mount_error.error_code() == mp::MountError::SSHFS_MISSING)
-            {
-                cmd::install_sshfs_for(mount_error.instance_name(), parser->verbosityLevel(), rpc_channel, stub, cout,
-                                       cerr);
-            }
-        }
+    auto on_failure = [this, &spinner](grpc::Status& status) {
+        spinner.stop();
 
-        return ret;
+        return standard_failure_handler_for(name(), cerr, status);
     };
 
+    auto streaming_callback = [&spinner](mp::MountReply& reply) { spinner.start(reply.mount_message()); };
```

Comment:
Shouldn't start be preceded by a stop for intermediate messages? I believe the spinner currently checks whether it is already running.

Reply:
Yeah, it won't hurt if it's not running and will address any intermediate messages as you say.

Reply:
Done